prisma/schema/Team.prisma

```diff
@@ -1,6 +1,6 @@
 model Team {
-  number      Int          @unique
-  name        String
-  team_event  TeamEvent[]
-  team_match  TeamMatch[]
+  number     Int         @unique
```

Comment:
I would call this field `key`
I know it would have to be changed in a bunch of places, sorry.